tutorials/README.rst

```diff
@@ -115,7 +115,7 @@ Notes on how to contribute more notebooks, and how the rendering is made, are at
      - `Yao-Yuan Mao <https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@yymao>`_
 
 
-   * - Truth Catalog with GCG
+   * - Truth Catalog with GCR
```

Comment:
Given that this ipynb is standalone, we want to make sure that folks know what things are, so I suggest adding (Generic Catalog Reader) to the first instance of GCR.